tests/internals.rs

```diff
@@ -265,12 +313,12 @@ fn test_declare_tx() {
 
     assert_eq!(
         fee_transfer_info.calldata,
-        vec![TEST_SEQUENCER_ADDRESS.clone(), 0.into(), 0.into()]
+        vec![TEST_SEQUENCER_ADDRESS.clone().0, 0.into(), 0.into()]
```

Comment:
Shouldn't matter in this case since it's a newtype struct, but cloning the smallest possible value should be more efficient than cloning something bigger and extracting part of it.
```suggestion
        vec![TEST_SEQUENCER_ADDRESS.0.clone(), Felt::zero(), Felt::zero()]
```